app/assets/stylesheets/components/_accordion.scss

```diff
@@ -16,6 +16,10 @@
 }
 
 .accordion {
+  border: 1px solid $border-color;
+  border-radius: $border-radius-md;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, .5);
```

Comment:
The style guide version I'm using (1.5) does have a box shadow around this component:

<img width="518" alt="screen shot 2017-04-14 at 12 58 49 pm" src="https://cloud.githubusercontent.com/assets/1421848/25049909/21c4a4ae-2112-11e7-972f-4e2a2418d9c1.png">

I think it might be clearer if it was removed, if it isn't meant to be used.